docs/v9/core/BrowserHistory.md

```
@@ -0,0 +1,99 @@
+# `BrowserHistory`
+
+Browser history stores the location in regular URLs. This is the standard for
+most web apps, but it requires some configuration on the server to ensure you
+serve the same app at multiple URLs.
+
+The full documentation for `BrowserHistory` can be found [here](https://github.com/remix-run/history/blob/main/docs/api-reference.md#history)
```

markdownlint-cli2 (0.23.2)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)